server/app/import.go

```diff
@@ -208,6 +209,9 @@ func (a *App) ImportBoardJSONL(r io.Reader, opt model.ImportArchiveOptions) (*mo
 					if err2 := json.Unmarshal(archiveLine.Data, &block); err2 != nil {
 						return nil, fmt.Errorf("invalid board block in archive line %d: %w", lineNum, err2)
 					}
+					if err := validateAttachmentIfPresent(block); err != nil {
```

Comment:
Can we include this logic as part of `block.IsValid()` and use it here? Maybe out of scope of this PR